test/InflationaryToken.test.ts

```diff
@@ -157,6 +162,7 @@ describe("InflationaryToken", () => {
         "InflationaryToken",
       );
       token = await InflationaryToken.connect(deployer).deploy(
+        cowDao.address,
```

Comment:
Same as before, even if unused I'd prefer:
```suggestion
        initialTokenHolder.address,
```